icecream/icecream.py

```diff
@@ -55,23 +56,13 @@ def colorize(s):
     return highlight(s, self.lexer, self.formatter)
 
 
-@contextmanager
-def supportTerminalColorsInWindows():
-    # Filter and replace ANSI escape sequences on Windows with equivalent Win32
-    # API calls. This code does nothing on non-Windows systems.
-    colorama.init()
-    yield
-    colorama.deinit()
```

Comment:
You could easily write this to still use a context manager and `deinit` after each print. Any reason not to?

Reply:
I spent a little time tweaking with the code base, trying to minimize code change, but I wasn't able to find a readable solution.

If we leave this contex manager in place, we should move conditional import into this context manager, thus we will try to import colorama on each call of `ic`. And `import` statement is a costly opertaion.

I've tried to move context manager into another module (`windows_support`), to be able to import `colorama` on a module level. But in that case we have to use conditional function definition such as:
```python

if sys.platform == 'win32':
    from .windows_support import supportTerminalColorsInWindows

    def colorizedStderrPrint(s):
        colored = colorize(s)
        with supportTerminalColorsInWindows():
            stderrPrint(colored)
else:
    def colorizedStderrPrint(s):
        colored = colorize(s)
        stderrPrint(colored)
```
In my opinion it comlicates both `icecream.py` and entrire package. So I had to throw away this idea.

After that I've read a little bit of colorama's ocumentation to find out the reason to `deinit` colorama after each call, but wasn't succesfful. So I wasn't sure is why do we need to call `init` and `deinit` in such matter.

So I've rewriten my proposition from ground up, and created this PR.

If you can point help me understand the reason behind this context manager, I will try to rewrite this PR again.

Reply:
See if you can do without init() and deinit() entirely using colorama.AnsiToWin32.

Reply:
I guess we can write:
```python

if sys.platform.startswith('win'):
    # We will import colorama only on a specific subset of OS's, and wrap sys.stderr into colorama's stream wrapper
    import colorama
    colorama.init(wrap=False)
    output_stream = colorama.AnsiToWin32(sys.stderr).stream
else:
    output_stream = sys.stderr

...

def stderrPrint(*args):
    print(*args, file=output_stream)
```

Sadly, in this case we have to rewrite `captureStandardStreams` context manager in  `test_icecream.py`, for it tries to replace stderr with fake. I tride to monkey-patch newly added module level attribute, but without any luck.